README.md

```diff
@@ -21,4 +21,17 @@ This project is currently under development. Use at your own risk. :)
   <p align="center">
     Description of PASEOS workflow on an individual device
   </p>
+</p>
+
+## Visualising PASEOS
+Navigate to paseos/visualization_example to find minimal examples of how to visualize PASEOS.
```

Comment:
I don't think we want the examples to be part of the module itself, otherwise they are also on pip, conda etc. Also, it is more error-prone because the paths will potentially be different if you are not in the module folder.

Let's move it one folder up into just examples/visualization/ ?